docs/src/groups.md

```diff
@@ -75,14 +75,17 @@ implementing the correct `IteratorSize` (i.e. `Base.HasLength()`, or
 optimized to work only based on the type of the group. In particular when the
 information is derivable from the type, there is no need to extend
 
-```@docs
-Base.isfinite(G::Group)
-```
-
 !!! note
     In the case that `IteratorSize(Gr) == IsInfinite()`, one should define
     `Base.length(Gr)` to be a "best effort", length of the group iterator.
 
     For practical reasons the largest group you could iterate over in your
     lifetime is of order that fits into an `Int`. For example, $2^{63}$
     nanoseconds comes to 290 years.
+
+## Additional methods
+
+```@docs
+Base.isfinite(G::Group)
+Base.istrivial(G::Group)
```

Comment:
There is no such method as `Base.istrivial`

Reply:
Indeed, silly me, fixed now